FiveCalls/FiveCalls/FetchIssuesOperation.swift

```diff
@@ -10,22 +10,35 @@ import Foundation
 
 class FetchIssuesOperation : BaseOperation {
     let zipCode: String?
-    
+    let latLongString: String?
+
     var httpResponse: HTTPURLResponse?
     
     var error: Error?
 
     var issuesList: IssuesList?
-    
+
     init(zipCode: String?) {
         self.zipCode = zipCode
+        self.latLongString = nil
+        super.init()
+    }
+    init(latLong: String?) {
+        self.zipCode = nil
+        self.latLongString = latLong
+        super.init()
     }
     
     lazy var sessionConfiguration = URLSessionConfiguration.default
     lazy var session: URLSession = { return URLSession(configuration: self.sessionConfiguration) }()
     
     override func execute() {
-        let url = URL(string: "https://5calls.org/issues/")!
+        var urlString = "https://5calls.org/issues/"
+        let urlQueryString: String? = zipCode ?? latLongString
+        if let query = urlQueryString {
+            urlString = urlString + "?address=" + query
```

Comment:
Yeah, though perhaps it would be better to break that into a separate ticket though with a thorough clarification of the fetch operation. @subdigital, what do you think?